scaling_performance/optimizing_storage.adoc

```diff
@@ -126,5 +126,22 @@ Overlay is also supported for Docker use cases as of Red Hat Enterprise Linux
 7.2, and provides faster start up time and page cache sharing, which can
 potentially improve density by reducing overall memory utilization.
 
-
-
+[[benefits-of-using-the-overlay-graph-driver]]
+== Benefits of Using the Overlay Graph Driver with SELinux
+
+The default Docker storage configuration on Red Hat Enterprise Linux (RHEL)
+continues to be device-mapper. As of RHEL 7.2, Overlay became a supported graph
```

Comment:
"continues to be a/the device-mapper"?
Also, do we need the hyphen in device-mapper?  Super quick Google search shows no hyphen.